client/client.go

```diff
@@ -105,6 +110,9 @@ func NewClient(baseDir string) (*Client, error) {
 			}
 		})
 	}
+	if c.EventLog = glow.NewEventLogger(EventLogExpiry, EventLogLimitBytes, EventLogLineLimitBytes); c.EventLog == nil {
```

Comment:
it would be better to have explicit error handling here, it's not considered good practice to return `nil` if an operation failed.